src/overlays/actors/ovl_Bg_Bdan_Objects/z_bg_bdan_objects.c

```diff
@@ -10,27 +10,38 @@
 
 #define FLAGS ACTOR_FLAG_4
 
+typedef enum {
+    GET_PROP_CAMERA_SETTING_NORMAL0 = 0,
+    GET_PROP_CAMERA_SETTING_DUNGEON1 = 3,
+    GET_PROP_WATCHED_BIGOCTO_INTRO_CUTSCENE = 4
+} BgBdanObjectPropertyGetter;
+typedef enum {
+    SET_PROP_CAMERA_SETTING_NORMAL0 = 1,
+    SET_PROP_CAMERA_SETTING_DUNGEON1 = 2,
+    SET_PROP_WATCHED_BIGOCTO_INTRO_CUTSCENE = 4
+} BgBdanObjectPropertySetter;
```

Comment:
Yeah im still not sure what to make of this weird system lmao

At the very least, we like prefix all enums with a namespace prefix to prevent possible name clashes. Even though obviously these names are hyper specific and would never clash, we just do it everywhere

You can use a "friendly" name for this prefix, so for example `JABU_OBJECTS_` would be fine.
You can take other shortening measures like changing "CAMERA" to "CAM" if you want

Reply:
Right? they could have at least used the same value for the getter/setter... maybe the fact that they didn't is some clue that I'm not understanding. Regardless, added the prefix 👍